src/app/components/Search/index.tsx

```diff
@@ -228,11 +231,10 @@ const SearchCmp: FC<SearchProps> = ({ scope, variant, disabled, onFocusChange: o
           },
         }}
         helperText={
-          value &&
-          value !== valueInSearchParams && (
-            <>
-              {!isTyping && hasError && (
-                <>
+          needsHelp && (
+            <Collapse in={needsHelp}>
```

Comment:
There is a reason why we are using _both_ as Collapse _and_ a condition. The Collapse only closes half-way, so to make it properly disappear, we also need the condition. Yes, it won't be visible all the time, but it still kinda works.

Reply:
Doesn't make sense to use Collapse here as animation will never work.

Reply:
```
        FormHelperTextProps={{
        // turn off TS temporary for next line 
        // issue https://github.com/mui/material-ui/issues/33339
        // fix https://github.com/mui/material-ui/pull/35924
          component: 'div', // replace p with div tag
          sx: {
            marginTop: 0, // set to zero  so we can remove `needsHelp &&`
            marginBottom: 0, // set to zero  so we can remove `needsHelp &&`
            marginLeft: variant === 'button' ? '48px' : '17px',
            marginRight: variant === 'button' ? '48px' : '17px',
          },
        }}
```

removed margin needs to be moved under the first collapse

```
        helperText={
          <Collapse in={needsHelp}>
            <Box sx={{ py: '10px' }}>
              <Collapse in={!isTyping && hasError}>
              ....
```

Reply:
That all works perfectly, thanks!